frontend/src/api/ErrorResponseHandler.ts

```diff
@@ -1,16 +1,16 @@
-import { AxiosError } from 'axios';
+import { AxiosError, AxiosResponse } from 'axios';
 import { ErrorResponse } from 'types/api';
 import { ErrorStatusCode } from 'types/common';
 
 export function ErrorResponseHandler(error: AxiosError): ErrorResponse {
-	const { response, request } = error;
+	const { response, request } = error as AxiosError;
```

Comment:
there is utility which tells whether object is AxiosError or not good to have rather than forcing